src/components/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { Button } from './Button';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const Header: React.FC<Props> = ({ todos }) => {
+  const [query, setQuery] = useState('');
+  const handleSubmitForm = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!query) {
+      return;
+    }
+
+    // const newTodo = {
+    //   id: +new Date(),
+    //   title: query,
+    //   completed: false,
+    // };
```

Comment:
This is needed to be removed too

Reply:
> This is needed to be removed too

Hi. Thx for review i alredy fix it in the second add-todo task. It would be great if u could approve  - so i can move forward